client/js/lounge.js

```diff
@@ -259,6 +259,16 @@ $(function() {
 		return msg;
 	}
 
+	// http://stackoverflow.com/a/1099670
+	function getQueryParam(name) {
+		var url = location.href;
```

Comment:
Use `location.search` instead to avoid the need to parse it. See http://stackoverflow.com/a/13419367/1739985 for a complete solution.